testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/drone/KeycloakDronePostSetup.java

```diff
@@ -56,7 +62,7 @@ public void configureWebDriver(@Observes AfterDroneEnhanced event, DroneContext
 
         if (drone instanceof RemoteWebDriver) {
             RemoteWebDriver remoteWebDriver = (RemoteWebDriver) drone;
```

Comment:
Since we now have a minimum version of Java 17 we can use [pattern matching for instanceof](https://openjdk.org/jeps/394) to avoid having to type-cast here.
```suggestion
        if (drone instanceof RemoteWebDriver remoteWebDriver) {
```